### ISSUE TITLE
[Classic Era] - GetContainerItemQuestInfo Error

### ISSUE THREAD

**Issue by Starlynk1** [doadin/Baggins#95]
Keep getting this error when opening the addon.
```
140x Interface\AddOns\Baggins\Baggins-v4.8.9.lua:3121: attempt to call upvalue 'GetContainerItemQuestInfo' (a nil value)
[string "@Interface\AddOns\Baggins\Baggins-v4.8.9.lua"]:3121: in function `UpdateItemButton'
[string "@Interface\AddOns\Baggins\Baggins-v4.8.9.lua"]:1921: in function `ReallyLayoutSection'
[string "@Interface\AddOns\Baggins\Baggins-v4.8.9.lua"]:762: in function `?'
[string "@Interface\AddOns\Baggins\Baggins-v4.8.9.lua"]:174: in function <Interface\AddOns\Baggins\Baggins.lua:166>

Locals:
self = <table> {
 cannotDetachTooltip = true
 modules = <table> {
 }
 CancelTimer = <function> defined @Interface\AddOns\Baggins\Libs\AceTimer-3.0\AceTimer-3.0.lua:147
 CreateItemButton = <function> defined @Interface\AddOns\Baggins\Baggins.lua:2514
 GetIncludeRule = <function> defined @Interface\AddOns\Baggins\Baggins-Filtering.lua:466
 RemoveSection = <function> defined @Interface\AddOns\Baggins\Baggins.lua:3908
 GetCachedItem = <function> defined @Interface\AddOns\Baggins\Baggins-Filtering.lua:538
 OptimizeSectionLayout = <function> defined @Interface\AddOns\Baggins\Baggins.lua:1569
 UpdateSectionContents = <function> defined @Interface\AddOns\Baggins\Baggins.lua:1200
 IsAnyBankOpen = <function> defined @Interface\AddOns\Baggins\Baggins.lua:4184
 RemoveRule = <function> defined @Interface\AddOns\Baggins\Baggins.lua:3925
 OnTextUpdate = <function> defined @Interface\AddOns\Baggins\Baggins.lua:3648
 InitOptions = <function> defined @Interface\AddOns\Baggins\Baggins-Options.lua:1082
 ScheduleTimer = <function> defined @Interface\AddOns\Baggins\Libs\AceTimer-3.0\AceTimer-3.0.lua:94
 UpdateDB = <function> defined @Interface\AddOns\Baggins\Baggins-Options.lua:1063
 GetBagTypes = <function> defined @Interface\AddOns\Baggins\Baggins-Filtering.lua:89
 UpdateItemButtons = <function> defined @Interface\AddOns\Baggins\Baggins.lua:2979
 LayoutSection = <function> defined @Interface\AddOns\Baggins\Baggins.lua:1859
 OnBankOpened = <function> defined @Interface\AddOns\Baggins\Baggins.lua:713
 SetEnabledState = <function> defined @Interface\AddOns\Baggins\Libs\AceAddon-3.0\AceAddon-3.0.lua:420
 Hook = <function> defined @Interface\AddOns\Baggins\Libs\AceHook-3.0\AceHook-3.0.lua:273
 RegisterSkin = <function> defined @Interface\AddOns\Baggins\Baggins-Skins.lua:34
 InitBagCategoryOptions = <function> defined @Interface\AddOns\Baggins\Baggins-Options.lua:3592
 UpdateLayout = <function> defined @Interface\AddOns\Baggins\Baggins.lua:2968
 CountNormalSlots = <function> defined @Interface\AddOns\Baggins\Baggins.lua:3736
 AddCustomRule = <function> defined @Interface\AddOns\Baggins\Baggins-Filtering.lua:101
 RuleTypeIterator = <function> defined @Interface\AddOns\Baggins\Baggins-Filtering.lua:123
 NewModule = <function> defined @Interface\AddOns\Baggins\Libs\AceAddon-3.0\AceAddon-3.0.lua:231
 MoveBag = <function> defined @Interface\AddOns\Baggins\Baggins.lua:3822
 ForceFullRefresh = <function> defined @Interface\AddOns\Baggins\Baggins-Filtering.lua:434
 DoRecheckCompress = <function> defined @Interface\AddOns\Baggins\Baggins-ItemOps.lua:230
 GetAce3Opts = <function> defined @Interface\AddOns\Baggins\Baggins-Filtering.lua:152
 GetName = <function> defined @Interface\AddOns\Baggins\Libs\AceAddon-3.0\AceAddon-3.0.lua:275
 MoveRule = <function> defined @Interface\AddOns\Baggins\Baggins.lua:3857
 defaultcategories = <table> {
 }
 ClearSectionCaches = <function> defined @Interface\AddOns\Baggins\Baggins.lua:999
 Disable = <function> defined @Interface\AddOns\Baggins\Libs\AceAddon-3.0\AceAddon-3.0.lua:310
 CategoryMatchRemoved = <function> defined @Interface\AddOns\Baggins\Baggins.lua:871
 Unhook = <function> defined @Interface\AddOns\Baggins\Libs\AceHook-3.0\AceHook-3.0.lua:417
 CloseSpecialWindows = <function> defined @Interface\AddOns\Baggins\Baggins.lua:4218
 CreateBagPlacementFrame = <function> defined @Interface\AddOns\Baggins\Baggins.lua:1978
 IncludeItemInCategory = <function> defined @Interface\AddOns\Baggins\Baggins-Filtering.lua:498
 Debug = <function> defined @Interface\AddOns\Baggins\Baggins.lua:117
 CategoryMatchAdded = <function> defined @Interface\AddOns\Baggins\Baggins.lua:811
 FireSignal = <function> defined @Interface\AddOns\Baggins\Baggins.lua:197
 IsActive = <function> defined @Interface\AddOns\Baggins\Baggins.lua:392
 CountAmmoSlots = <function> defined @Interface\AddOns\Baggins\Baggins.lua:3740
 SlotMoved = <function> defined @Interface\AddOns\Baggins\Baggins.lua:910
 SetDefaultModuleLibraries = <function> defined @Interface\AddOns\Baggins\Libs\AceAddon-3.0\AceAddon-3.0.lua:363
 Baggins_CategoriesChanged = <function> defined 
```

**Comment by Starlynk1** [comment]
I found a workaround to get the errors away. Classic Era does not use the GetContainerItemQuestInfo(bag, slot) since that was not added until Wrath. I edited line 3121 as such:
```
        isQuestItem, questId, isActive = nil --GetContainerItemQuestInfo(bag, slot)
```
Now I just need to figure out how to remove the Quest window and all is golden.

**Comment by Starlynk1** [comment]
Also made a change to the *New* text for items. It was a bit too big, so made it smaller and changed the shadow so it looks better in game.
```
        frame.newtext:SetShadowOffset(-1,-1)
		frame.newtext:SetFont(LSM and LSM:Fetch("font", Baggins.db.profile.Font) or _G.STANDARD_TEXT_FONT,Baggins.db.profile.FontSize or 10)
```

**Comment by doadin** [comment]
> I found a workaround to get the errors away. Classic Era does not use the GetContainerItemQuestInfo(bag, slot) since that was not added until Wrath. I edited line 3121 as such:
> 
> ```
>         isQuestItem, questId, isActive = nil --GetContainerItemQuestInfo(bag, slot)
> ```
> 
> Now I just need to figure out how to remove the Quest window and all is golden. (Just used the Hide Empty bag option and won't see it ever again)

Fixed the error in v4.9.0.
